clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileStatus.java

```diff
@@ -0,0 +1,34 @@
+package io.lakefs;
+
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.Path;
+
+import javax.annotation.Nullable;
+import java.util.Objects;
+
```

Comment:
Please document when we return this status.  I think our users should be allowed to depend on it; if you agree then we should document it.